pkg/capr/installer/server_test.go

```diff
@@ -42,6 +43,11 @@ func TestHandler_ServeHTTP(t *testing.T) {
 			handler := handler{}
 
 			// act
+			err = settings.SystemAgentInstallScript.Set("https://raw.githubusercontent.com/rancher/system-agent/main/install.sh")
```

Comment:
Should we pick something more stable than `main` for this?

Reply:
Without turning it into a burden, I don't think so. 

If we ever move away from the `main` branch having a functional `install.sh` then we can deal with it then. 

the techdebt here would be to move the source of truth for this script somewhere that can be easily accessed